corrector.rb

```diff
@@ -0,0 +1,10 @@
+# Define Corrector class
+class Corrector
+  def initialize(name)
+    @name = name
+  end
+
+  def correct_name
+    @name.capitalize[0, 9]
+  end
```

Comment:
- [ ] The name must have a maximum of 10 characters. If it's longer it should trim the word. But here you are considering only the name up to 9 characters. Please update this. [Hint: `@name.capitalize[0, 10]`]